test/test_models.py

```diff
@@ -38,65 +38,72 @@ def get_available_video_models():
 # models that are in torch hub, as well as r3d_18. we tried testing all models
 # but the test was too slow. not included are detection models, because
 # they are not yet supported in JIT.
-script_test_models = [
-    "deeplabv3_resnet101",
-    "mobilenet_v2",
-    "resnext50_32x4d",
-    "fcn_resnet101",
-    "googlenet",
-    "densenet121",
-    "resnet18",
-    "alexnet",
-    "shufflenet_v2_x1_0",
-    "squeezenet1_0",
-    "vgg11",
-    "inception_v3",
-    'r3d_18',
+# If 'unwrapper' is provided it will be called with the script model outputs
+# before they are compared to the eager model outputs. This is useful if the
+# model outputs are different between TorchScript / Eager mode
+script_test_models = {
+    "deeplabv3_resnet101": {},
+    "mobilenet_v2": {},
+    "resnext50_32x4d": {},
+    "fcn_resnet101": {},
+    "googlenet": {
+        "unwrapper": lambda x: x.logits
+    },
+    "densenet121": {},
+    "resnet18": {},
+    "alexnet": {},
+    "shufflenet_v2_x1_0": {},
+    "squeezenet1_0": {},
+    "vgg11": {},
+    "inception_v3": {},
```

Comment:
I think inception also need to be unwrapped https://github.com/pytorch/vision/blob/79daca1027182361bc9d8322d04838cbc4196348/torchvision/models/inception.py#L161-L166 but on the script output